src/components/VideoCard/VideoCard.js

```diff
@@ -20,7 +43,31 @@ export default function VideoCard({ video, selectOptions }) {
         >
           <ul className={`options ${showOptions ? "showOptions" : ""}`}>
             {selectOptions.map((list) => (
-              <li key={Math.random()} onClick={() => setShowOptions(true)}>
+              <li
+                key={Math.random()}
```

Comment:
Generating key using Math.random() is not a good practice. Every time a component's key changes , a new component instance will be created, rather than updating the existing one, which will be sub-optimal with respect to performance of the app.

Reply:
Thank you for the suggestions. I will fix this.